curvefs/src/tools/README.md

```diff
@@ -0,0 +1,501 @@
+curvefs_tool status-mds
+```
+
+Output:
+
+```shell
+mds version: 1.0.0
+leader mds: 10.182.2.45:16700
+standy mds: [ ].
+offline mds: [ 10.182.2.45:27700 10.182.2.45:36700 ].
+```
+
+### **status-metaserver**
+
+show the status of metaserver who are specified by metaserverAddr in the configuration file
+
+Usage:
+
+```shell
+curvefs_tool status-metaserver
+```
+
+Output:
+
+```shell
+metaserver version: 1.0.0
+online metaserver: [ 10.182.2.45:16701 10.182.2.45:36701 ].
+offline metaserver: [ 10.182.2.45:26701 ].
+```
+
+### **status-etcd**
+
+show the status of metaserver who are specified by etcdAddr in the configuration file
+
+Usage:
+
+```shell
+curvefs_tool status-etcd
+```
+
+Output:
+
+```shell
+etcd version: 3.4.0
+leader etcd: 10.182.2.45:12379
+standy etcd: [ 10.182.2.45:22379 ].
+offline etcd: [ 10.182.2.45:32379 ].
+```
+
+### **status-copyset**
+
+show the status of copyset
+
+Usage:
+
+```shell
+curvefs_tool status-copyset
+```
+
+Output:
+
+```shell
+all copyset is health.
+copyset[4294967297]:
+-info:
+statusCode: TOPO_OK copysetInfo { poolId: 1 copysetId: 1 peers { id: 1 address: "10.182.2.45:36701:0" } peers { id: 2 address: "10.182.2.45:26701:0" } peers { id: 3 address: "10.182.2.45:16701:0" } epoch: 0 leaderPeer { id: 3 address: "10.182.2.45:16701:0" } }
+-status:
+status: COPYSET_OP_STATUS_SUCCESS copysetStatus { state: 1 peer { address: "10.182.2.45:16701:0" } leader { address: "10.182.2.45:16701:0" } readonly: false term: 2 committedIndex: 1 knownAppliedIndex: 1 pendingIndex: 0 pendingQueueSize: 0 applyingIndex: 0 firstIndex: 1 lastIndex: 1 diskIndex: 1 epoch: 0 }
+status: COPYSET_OP_STATUS_SUCCESS copysetStatus { state: 4 peer { address: "10.182.2.45:26701:0" } leader { address: "10.182.2.45:16701:0" } readonly: false term: 2 committedIndex: 1 knownAppliedIndex: 1 pendingIndex: 0 pendingQueueSize: 0 applyingIndex: 0 firstIndex: 1 lastIndex: 1 diskIndex: 1 epoch: 0 }
+status: COPYSET_OP_STATUS_SUCCESS copysetStatus { state: 4 peer { address: "10.182.2.45:36701:0" } leader { address: "10.182.2.45:16701:0" } readonly: false term: 2 committedIndex: 1 knownAppliedIndex: 1 pendingIndex: 0 pendingQueueSize: 0 applyingIndex: 0 firstIndex: 1 lastIndex: 1 diskIndex: 1 epoch: 0 }
+
+...
+
+```
+
+[Back to Toc](#Contents)
+
+---
+
+## list
+
+### **list-fs**
+
+list all fs in cluster
+
+Usage:
+
+```shell
+curvefs_tool list-fs
+```
+
+Output:
+
+```shell
+fsId: 1
+fsName: "/test"
+status: INITED
+rootInodeId: 1
+capacity: 18446744073709551615
+blockSize: 1
+mountNum: 1
+mountpoints: "09a03e7f5ece:/usr/local/curvefs/client/mnt"
+fsType: TYPE_S3
+detail {
+  s3Info {
+    ak: "********************************"
+    sk: "********************************"
+    endpoint: "********************************"
+    bucketname: "********************************"
+    blockSize: 1048576
+    chunkSize: 4194304
+  }
+}
+
+...
+
+```
+
+### **list-coysetInfo**
+
+list all copysetInfo in cluster
+
+Usage:
+
+```shell
+curvefs_tool list-copysetInfo
```

Comment:
Maybe list all the copysets on a metaserver is needed.

List command also need to add 'list server; list metaserver; list partition; list partiton of fs and so on'.

Reply:
> 

will add.